1/step1.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+        value_to_diff = {}
+        for value in nums:
+            value_to_diff[value] = target - value
+
+        sorted_nums = sorted(nums)
+        answer1 = 0
+        answer2 = 0
+        for index in range(len(nums)):
```

Comment:
len(sorted_nums) = len(nums)なので問題は無いのですが、numsに対するrangeで回しているのに中ではsorted_numsにアクセスしているというズレが気になりました。
また、このループの中ではindex自体を使わないので以下で良いかなと思いました。

```suggestion
        for num in sorted_nums:
```